docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -27,7 +29,7 @@ Batching is a straightforward way of leveraging the GPU compute power and saving
 @endsphinxtabset
 
 
-> **NOTE**: You can disable the Auto-Batching (for example, for the GPU device) from being triggered by the `ov::hint::PerformanceMode::THROUGHPUT`. To do that, pass the `ov::hint::allow_auto_batching` set to **false** in addition to the `ov::hint::performance_mode`:
+> **NOTE**: The Auto-Batching can be disabled (for example, for the GPU device) from being triggered by the `ov::hint::PerformanceMode::THROUGHPUT`. To do that, pass the `ov::hint::allow_auto_batching` set to **false** in addition to the `ov::hint::performance_mode`:
```

Comment:
```suggestion
> **NOTE**: Auto-Batching can be disabled (for example, for the GPU device) to prevent being triggered by `ov::hint::PerformanceMode::THROUGHPUT`. To do that, set `ov::hint::allow_auto_batching` to **false** in addition to the `ov::hint::performance_mode`, as shown below:
```

Reply:
We can move the description of this note and the following code example to a ### Disabling Auto-Batching heading at the end of the "## Enabling Automatic Batching" section, to avoid breaking the content logic.